### PR TITLE
Add Rancher metadata to swarm templates

### PR DESCRIPTION
**Pull request by LLParse**
Fixes https://github.com/rancher/rancher/issues/4101

### CODE REVIEW

File: swarm-templates/wordpress/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      label: "Client Port"
+      required: true
+      default: "80"
+      type: "int"
+    - variable: db_password
+      description: "Password for database"
+      label: "Password"
+      required: true
+      default: "wordpress"
+      type: "string"
```
Comment (cloudnautique) [inline swarm-templates/wordpress/0/rancher-compose.yml]: type should be password

File: swarm-templates/wordpress/config.yml [hunk 2]
```diff
@@ -3,3 +3,4 @@ description: |
   Blog tool, publishing platform and CMS
 version: v0.1-educaas1
 category: Blog
+
```
Comment (cloudnautique) [inline swarm-templates/wordpress/config.yml]: no white space additions :)

File: swarm-templates/cassandra/0/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.catalog:
+  name: "Cassandra"
+  version: "v0.1a"
```
Comment (cloudnautique) [inline swarm-templates/cassandra/0/rancher-compose.yml]: this needs to match config.yml which says 1.0.0

File: swarm-templates/ghost/0/rancher-compose.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+.catalog:
+  name: "Ghost"
+  version: "v0.7.8-rancher1"
```
Comment (cloudnautique) [inline swarm-templates/ghost/0/rancher-compose.yml]: needs to match version in config.yml

**Comment by LLParse** [comment]
@cloudnautique I synced up the versions for the templates.

We should start thinking about a formal versioning strategy (maybe after RC)

**Comment by cloudnautique** [comment]
LGTM. Thanks.

**Pull request merged**